lang/en.json

```diff
@@ -0,0 +1,76 @@
+{
```

Comment:
Generally, I think we should favor using DND5e's localized strings over creating new ones. For instance, DND5e has 

```json
"DND5E.ConBlinded": "Blinded"
```

among some other ones. It does not do any spells though.

Reply:
Basically what I'm saying is if it is defined in dnd5e, just use that. Otherwise, create a new one.